packages/i18n/README.md

```diff
@@ -0,0 +1,198 @@
+# I18n Package
+
+This package provides a set of tools to help you internationalize your Chrome Extension.
+
+https://developer.chrome.com/docs/extensions/reference/api/i18n
+
+## Installation
+
+If you want to use the i18n translation function in each pages, you need to add the following to the package.json file.
+
+```json
+{
+  "dependencies": {
+    "@extension/i18n": "workspace:*"
+  }
+}
+```
+
+Then run the following command to install the package.
+
+```bash
+pnpm install
+```
+
+## Manage translations
+
+You can manage translations in the `locales` directory.
+
+`locales/en/messages.json`
+
+```json
+{
+  "helloWorld": {
+    "message": "Hello, World!"
+  }
+}
+```
+
+`locales/ko/messages.json`
+
+```json
+{
+  "helloWorld": {
+    "message": "안녕하세요, 여러분!"
+  }
+}
+```
+
+If you want to use placeholders, you can use the following format.
+
+https://developer.chrome.com/docs/extensions/how-to/ui/localization-message-formats#placeholders
+
+> [!Caution]
+> You should use placeholders in this format: `$1`, `$2`, `$3`, ... $/d+
+> Still, We don't support the placeholders field in the `messages.json` file.
```

Comment:
Maybe, implement it also, on this PR?
I seeing you have 'TODO' in 'i18n-dev' file

Reply:
I'll look into the way placeholders work a bit more and refine it so we can implement it all at once~!

Reply:
I've been looking for a library that implements (or at least mimics) Chrome's i18n feature and I can't find one. I've looked at Chromium's source code, but there's only test code for the i18n feature, so it's hard to reference a detailed implementation...

https://chromium.googlesource.com/chromium/src/+/901414aca5338840519c9fa7ee3fb82cb4f0f18f/chrome/common/extensions/api/extension_api.json
https://chromium.googlesource.com/chromium/src/+/901414aca5338840519c9fa7ee3fb82cb4f0f18f/chrome/common/extensions
https://github.com/chromium/chromium/blob/main/chrome/browser/extensions/api/i18n/i18n_apitest.cc

Reply:
https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite/pull/613/commits/8abc4f6f07aba0b2912a68f5339b9d33e98b7350

I think this is a implementation for chrom.i18n API.